test/tests/interfaces/pbs_sched_interface_test.py

```diff
@@ -55,28 +53,7 @@ def setUp(self):
         self.server.manager(MGR_CMD_CREATE,
                             SCHED, a,
                             id="TestCommonSched")
-        self.server.manager(MGR_CMD_SET, SCHED,
-                            {'sched_port': 15051},
-                            id="TestCommonSched")
-        self.sched_configure("TestCommonSched", '15051')
-
-    def sched_configure(self, sched_name, sched_port, sched_home=None):
-        pbs_home = self.server.pbs_conf['PBS_HOME']
-        if sched_home is None:
-            sched_home = pbs_home
-        sched_priv_dir = 'sched_priv_' + sched_name
-        sched_logs_dir = 'sched_logs_' + sched_name
-        if not os.path.exists(os.path.join(sched_home, sched_priv_dir)):
-            self.du.run_copy(self.server.hostname,
-                             os.path.join(pbs_home, 'sched_priv'),
-                             os.path.join(sched_home, sched_priv_dir),
-                             recursive=True)
-        if not os.path.exists(os.path.join(sched_home, sched_logs_dir)):
-            self.du.run_copy(self.server.hostname,
-                             os.path.join(pbs_home, 'sched_logs'),
-                             os.path.join(sched_home, sched_logs_dir),
-                             recursive=True)
-        self.server.schedulers[sched_name].start(sched_port, sched_home)
+        self.sched_configure("TestCommonSched")
```

Comment:
You're calling sched_configure(), but you just removed the function.

Reply:
I have added it in pbs_testsuite.py as per varun's suggestion as i am using it in both functional and interface test suites .

Reply:
I disagree with @varunsonkar.  While duplicating code is bad, putting test specific functions in the parent class to all tests is worse.  The parent class should remain generic to all tests.

Reply:
Hi @bhroam , 
To start all the schedulers even in future we need to create a priv directory and log directory , Before starting the scheduler . So i had written this function . It can used in future in all the test suites which might need one or more scheduler . Wont keeping it in pbs_testsuite.py help ?

Reply:
This function does sound useful.  I think you should modify it some and put it as part of the scheduler object in PTL itself.  Maybe instead of sched_configure(), you write create_scheduler().  You pass in the name of the scheduler and create a working scheduler.  I'd also not do a copy of the sched_log directory.  Just make the directory.  You don't want to start matching logs from the default scheduler.